mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,8 +1,12 @@
 CLASS net/minecraft/unmapped/C_pjtstjoq net/minecraft/inventory/Inventory
 	FIELD f_afhsixex MAX_COUNT_PER_STACK I
+	FIELD f_vlssyyaw DEFAULT_MAX_INTERACTION_RANGE F
 	METHOD m_akaynoad setStack (ILnet/minecraft/unmapped/C_sddaxwyk;)V
 		ARG 1 slot
 		ARG 2 stack
+	METHOD m_crvopqlx canPlayerUse (Lnet/minecraft/unmapped/C_kvegafmh;Lnet/minecraft/unmapped/C_jzrpycqo;F)Z
+		ARG 0 entity
+		ARG 2 range
```

Comment:
This is more like an added range, it looks to be in addition to the base player interaction range.
```suggestion
		ARG 2 additionalRange
```